### ISSUE TITLE
Participants userphotos and usernames in conversations are displayed in seperate rows

### ISSUE THREAD

**Issue by 4311021x** [kasperisager/vanilla-bootstrap#149]
Hi Kasper,

thanks for your great work on this theme!

I have noticed that participants userphotos and usernames in conversations are displayed in separate rows and I have worked out a fix for yeti already. I was just about to create a branch and offer a pull request but I am not very familiar with less so here you are:

~~~
.InThisConversation .PanelInfo li {
  position: relative;
  display: flex;
  background-color: #FFF;
  border: 1px solid #DDD;
}

.PanelInfo a.PhotoWrap {
  flex: 1;
  border: 0
}

.PanelInfo a.UserLink {
  flex: 23;
  border: 0
}
~~~

Kind regards

Hendrik

**Comment by joseamirandavelez** [comment]
Where did you put this? In the custom.css file? It didnt work for me.

On Sat, Jan 31, 2015 at 7:31 PM, 4311021x <notifications@github.com> wrote:

> Hi Kasper,
>
> thanks for your great work on this theme!
>
> I have noticed that participants userphotos and usernames in conversations
> are displayed in separate rows and I have worked out a fix already. I was
> just about to create a branch and offer a pull request but I am not very
> familiar with less so here you are:
>
> .InThisConversation .PanelInfo li {
>   position: relative;
>   display: flex;
>   background-color: #FFF;
>   border: 1px solid #DDD;
> }
>
> .PanelInfo a.PhotoWrap {
>   flex: 1;
>   border: 0
> }
>
> .PanelInfo a.UserLink {
>   flex: 23;
>   border: 0
> }
>
> Kind regards
>
> Hendrik
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kasperisager/vanilla-bootstrap/issues/149>.
>



-- 
- Jose Miranda

**Comment by joseamirandavelez** [comment]
can you please send a link to your forum?

On Sat, Jan 31, 2015 at 10:30 PM, Jose Miranda <joseamirandavelez@gmail.com>
wrote:

> Where did you put this? In the custom.css file? It didnt work for me.
>
> On Sat, Jan 31, 2015 at 7:31 PM, 4311021x <notifications@github.com>
> wrote:
>
>> Hi Kasper,
>>
>> thanks for your great work on this theme!
>>
>> I have noticed that participants userphotos and usernames in
>> conversations are displayed in separate rows and I have worked out a fix
>> already. I was just about to create a branch and offer a pull request but I
>> am not very familiar with less so here you are:
>>
>> .InThisConversation .PanelInfo li {
>>   position: relative;
>>   display: flex;
>>   background-color: #FFF;
>>   border: 1px solid #DDD;
>> }
>>
>> .PanelInfo a.PhotoWrap {
>>   flex: 1;
>>   border: 0
>> }
>>
>> .PanelInfo a.UserLink {
>>   flex: 23;
>>   border: 0
>> }
>>
>> Kind regards
>>
>> Hendrik
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/kasperisager/vanilla-bootstrap/issues/149>.
>>
>
>
>
> --
> - Jose Miranda
>



-- 
- Jose Miranda

**Comment by kasperisager** [comment]
Thanks for the ups! The issue you're seeing has been fixed in Vanilla core and was due to outdated markup in the Conversations application: vanilla/vanilla#1947

**Comment by 4311021x** [comment]
I'm not sure if I get that right. If it's fixed since 2.1.1 (because its in milestone 2.1.1), why do I see this issue in 2.1.8p2?

@joseamirandavelez I've made a plugin to hook in and change some theming stuff without touching the original files to avoid additional work when updating but this snippet should also work in the custom.css file. I've tested it with the latest Firefox, IE & Chrome. A forum link wouldn't be helpful because you would have to be logged in to test it and I may not create an account for you. If you give me a link to yours I could take a look why its not working thought.
